code/game/machinery/alarm.dm

```diff
@@ -976,13 +989,13 @@ var/global/list/air_alarms = list()
 		else
 			max_temperature = temperature_threshold.max_1() - T0C
 			min_temperature = temperature_threshold.min_1() - T0C
-		var/input_temperature = input("What temperature (in C) would you like the system to target? (Capped between [min_temperature]C and [max_temperature]C).\n\nNote that the cooling unit in this air alarm can not go below [MIN_TEMPERATURE]C or above [MAX_TEMPERATURE]C by itself. ", "Thermostat Controls") as num|null
+		var/input_temperature = input("What temperature (in C) would you like the system to target? (Capped between [min_temperature]C and [max_temperature]C).\n\nNote that the cooling unit in this air alarm can not go below [MIN_TEMPERATURE - T0C]C or above [MAX_TEMPERATURE - T0C]C by itself. ", "Thermostat Controls") as num|null
```

Comment:
```suggestion
		var/input_temperature = input("What temperature (in C) would you like the system to target? (Capped between [min_temperature]C and [max_temperature]C).\n\nNote that the cooling unit in this air alarm can not go below [MIN_TEMPERATURE - T0C]C or above [MAX_TEMPERATURE + T0C]C by itself. ", "Thermostat Controls") as num|null
```

Reply:
```
#define MAX_TEMPERATURE T0C+90
#define MIN_TEMPERATURE T0C-40
```
Which is 90C and -40C.
So, to confirm that the math is right...
(90 + T0C) = 90C (in K)
(-40 + T0C) = -40C (in K)
Adding T0C to a C number converts it to K. So, to convert from K to C, you must subtract T0C as follows:
(-40 + T0C) - T0C = -40C
(90 + T0C) - T0C = 90C
vs the proposed change,
(90 + T0C) + T0C = considerably higher than 90C

Therefore the original line is correct.